### PR TITLE
Updated translatedLibraryMap to store versionedIdentifier.getId() as …

### PR DESCRIPTION
**Pull request by RohitKandimalla**
…a string

## CQL to ELM Translation Service PR

Jira Ticket: [MAT-6312](https://jira.cms.gov/browse/MAT-6312)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/mat/cql_elm_translation/utils/cql/CQLTools.java [hunk 1]
```diff
@@ -79,26 +79,26 @@ public class CQLTools {
...
+  //      List<String> parentExpressions,
+  //      CqlTranslator translator) {
+  //
+  //    this(
+  //        parentLibraryString,
+  //        childrenLibraries,
+  //        parentExpressions,
+  //        translator,
+  //        translator.getTranslatedLibraries());
+  //  }
```
Comment (jkotanchik-SB) [inline src/main/java/gov/cms/mat/cql_elm_translation/utils/cql/CQLTools.java]: Remove dead code.

**Review by jkotanchik-SB — DISMISSED** [review]

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@ info:
 madie:
   library:
     service:
-      baseUrl: ${CQL_LIBRARY_SERVICE_URL:http://localhost:8082/api}
```
Comment (nmorasb) [inline src/main/resources/application.yaml]: do we need to change the port?

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ logging:
 
 okta:
   oauth2:
-    issuer: ${OKTA_ISSUER:https://dev-18092578.okta.com/oauth2/default}
```
Comment (nmorasb) [inline src/main/resources/application.yaml]: do we need to change the default okta instance?

Comment (RohitKandimalla) [reply src/main/resources/application.yaml]: that was a mistake @nmorasb I reverted it. Thanks

**Review by jkotanchik-SB — APPROVED** [review]

**Review by nmorasb — APPROVED** [review]

**Pull request merged**